orbit/utils/docstring_style.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+
+from collections import OrderedDict
+from inspect import cleandoc
+from custom_inherit._doc_parse_tools.numpy_parse_tools import \
+        parse_numpy_doc
+
+__all__ = ["merge_numpy_docs"]
+
+
+def merge_section(key, prnt_sec, child_sec, merge_within_sections=False):
```

Comment:
it seems missing a docstring for argumennt `merge_within_sections`

Reply:
added corresponding docstring